docs/references/chrome-extension/sync-host.mdx

```diff
@@ -91,6 +91,36 @@ Clerk allows you to sync the authentication state from your web app to your Chro
   }
   ```
 
+  ### Hide unsupported authentication methods
+
+  When using the Sync Host feature you may want to enable authentication methods on the web application that are unsupported in the extension. The Dashboard settings affect anything using the Clerk application, so those methods would show up in your Chrome Extension's sign up and sign in components. Below you will find options to hide unsupported methods.
+
+  <CodeBlockTabs options={["Sign-up", "Sign-in"]}>
+    ```tsx
+    <SignUp
+      routing="virtual"
```

Comment:
@royanger is `routing="virtual"` needed here? There was a bit of discussion around `virtual` nooping when not within a modal here today https://clerkinc.slack.com/archives/C079PMLTDLK/p1733954224937959

Reply:
Good callout @alexcarpenter. I'm pretty sure that it was. needed with React Router some time ago, but I will test and see if it still is. I think it may not be.